### PR TITLE
Front : user profile

### PR DESCRIPTION
**Pull request by AdrienMazet**

### CODE REVIEW

File: packages/front/src/containers/profile/ProfileForm.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+          email={profile.email}
+          joinDate={profile.joinDate}
+          editProfile={handleEditProfile}
+          deleteProfile={handleDeleteProfile}
+          disabled={formDisabled}
+        />
+      )}
+      {error && (
+        <Alert
+          message="An error occured"
```
Comment (rigwild) [inline packages/front/src/containers/profile/ProfileForm.tsx]: `occurred`* (also in `WebcamRecorder.tsx`)

File: packages/front/src/components/profile/ProfileForm.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        </Paragraph>
+      </div>
+      <div style={labelContainerStyle}>
+        <Text strong>Email address :</Text>
+        <Paragraph style={{ marginLeft: 15 }}>{props.email}</Paragraph>
+      </div>
+      <div style={labelContainerStyle}>
+        <Text strong>Join date :</Text>
+        <Paragraph style={{ marginLeft: 15 }}>
+          {new Date(props.joinDate).toLocaleDateString()}
```
Comment (rigwild) [inline packages/front/src/components/profile/ProfileForm.tsx]: A hack to get `yyyy-mm-dd` easily:
```js
new Date().toLocaleDateString('en-CA')
// -> "2020-05-30"
```

**Review by rigwild — CHANGES_REQUESTED** [review]
Nice 😊

File: packages/front/src/containers/profile/ProfileForm.tsx (hunk 1 above)
Comment (AdrienMazet) [reply packages/front/src/containers/profile/ProfileForm.tsx]: 4a44aa0

**Review by rigwild — APPROVED** [review]

**Pull request merged**